src/options/options.ts

```diff
@@ -103,24 +99,12 @@ export class UIPOptions extends UIPPlugin {
     }
   }
 
-  protected changeEditorTheme(theme: string) {
-    const $editor = this.root?.querySelector(`:scope > ${UIPEditor.is}`) as UIPEditor;
-    const editorConfig = $editor?.editorConfig;
-    if (!$editor || !editorConfig) return;
-
-    editorConfig.theme = theme === 'uip-dark' ? UIPOptions.darkEditorTheme : UIPOptions.lightEditorTheme;
-    $editor.setEditorConfig(editorConfig);
-  }
-
   protected updateModeMarker(mode: string) {
-    this.root && CSSClassUtils.remove(this.root, 'vertical-mode horizontal-mode');
-    this.root && CSSClassUtils.add(this.root, `${mode}-mode`);
+    this.root && this.root.setAttribute('mode', `${mode}`);
```

Comment:
Propose to create the property accessor on the UIPRoot level.
PS: use `@attr decorator to do that`